pysatModels/models/ucar_tiegcm.py

```diff
@@ -32,17 +40,19 @@
 
 platform = 'ucar'
 name = 'tiegcm'
-tags = {'': 'UCAR TIE-GCM file'}
-inst_ids = {'': ['']}
+tags = {'': 'UCAR TIE-GCM file', 'icon': 'UCAR TIE-GCM file produced for ICON.'}
+inst_ids = {'': list(tags.keys())}
 
-# specify using xarray (not using pandas)
+# Specify using xarray (not using pandas)
 pandas_format = False
 
 # ----------------------------------------------------------------------------
 # Instrument test attributes
 
-_test_dates = {'': {'': dt.datetime(2019, 1, 1)}}
-_test_download = {'': {'': False}}
+_test_dates = {'': {'': dt.datetime(2019, 1, 1),
+                    'icon': dt.datetime(2020, 1, 10)}}
+_test_download = {'': {'': False, 'icon': True}}
+_test_download_ci = {'': {'': False, 'icon': True}}
```

Comment:
I don't think these are the settings you want.  My understanding is that we have no download support for the default form, and we want to skip icon on the ci system (since the files are huge).  This would be:
```suggestion
_test_download = {'': {'': False}}
_test_download_ci = {'': {'icon': False}}
```
If you set the ci flag to False, then the default checks for the warning routine will be skipped in the core code because we are skipping download tests only in a ci environment, but want to run them locally.

If we are not skipping the icon downloads on github actions, then remove the `_test_download_ci` flag completely

Reply:
Looking through the logic in the test file, it still shouldn't get the @download mark if either of those is set.  Not sure why the remote_file tests are being run, those should be skipped for default

Reply:
resolving in favor of alternate solution